verticapy/performance/vertica/collection/profile_import.py

```diff
@@ -141,11 +210,92 @@ def _create_schema_if_not_exists(self) -> None:
 
     def _create_tables_if_not_exists(self) -> None:
         all_tables = getAllCollectionTables(
-            target_schema=self.target_schema, key=self.key
+            target_schema=self.target_schema, key=self.key, version=self.bundle_version
         )
         for ctable in all_tables.values():
             self.logger.info(f"Running create statements for {ctable.name}")
             table_sql = ctable.get_create_table_sql()
             proj_sql = ctable.get_create_projection_sql()
             _executeSQL(table_sql, method="fetchall")
             _executeSQL(proj_sql, method="fetchall")
+
+    def _unpack_bundle(self) -> Path:
+        """
+        Unpacks the bundle into a temp directory
+
+        Returns
+        --------
+        Returns a path
+        """
+
+        self.tarfile_obj = tarfile.open(self.filename, "r")
+        names = "\n".join(self.tarfile_obj.getnames())
+        print(f"Files in the archive: {names}")
+        # There are other ways to generate a
+        tmp_dir = self.tmp_path / Path(
+            f"profile_import_run{random.randint(10000, 20000)}"
+        )
+
+        print(f"Creating temporary directory: {tmp_dir}")
+
+        tmp_dir.mkdir()
+
+        self.tarfile_obj.extractall(tmp_dir)
+
+        print(f"Extracted files: {[x for x in tmp_dir.iterdir()]}")
+
+        return tmp_dir
+
+    def _calculate_bundle_version(self, unpack_dir: Path) -> BundleVersion:
```

Comment:
A quick description is needed to explain what you do.

Reply:
added a docstring comment